### ISSUE TITLE
MDAnalysis breaks on import

### ISSUE THREAD

**Issue by PasinduT** [MDAnalysis/mdanalysis#3087]
## MDAnalysis breaks on import

This is because of Biopython version 1.78 removing Alphabet. See error below:

```python
ImportError: Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://biopython.org/wiki/Alphabet for more information.
```
Downgrading to Biopython 1.77 works.

**Comment by IAlibay** [comment]
@PasinduT would you be able to provide more information about your MDAnalysis install? 

Particularly, what version are you using?

**Comment by PasinduT** [comment]
Sure. This is the version of my MDAnalysis and Biopython.
```
# Name                    Version                   Build  Channel
biopython                 1.78             py37h4abf009_1    conda-forge
mdanalysis                0.20.1           py37he1b5a44_0    conda-forge
```

And here is the full error
```python
In [1]: import MDAnalysis                                                                                                                                                   
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-4c4262a6b4c9> in <module>
----> 1 import MDAnalysis

~/anaconda3/lib/python3.7/site-packages/MDAnalysis/__init__.py in <module>
    182 )
    183 
--> 184 from .lib import log
    185 from .lib.log import start_logging, stop_logging
    186 

~/anaconda3/lib/python3.7/site-packages/MDAnalysis/lib/__init__.py in <module>
     38 from . import mdamath
     39 from . import distances  # distances relies on mdamath
---> 40 from . import NeighborSearch
     41 from . import formats
     42 from . import pkdtree

~/anaconda3/lib/python3.7/site-packages/MDAnalysis/lib/NeighborSearch.py in <module>
     35 from MDAnalysis.lib.util import unique_int_1d
     36 
---> 37 from MDAnalysis.core.groups import AtomGroup, Atom
     38 
     39 

~/anaconda3/lib/python3.7/site-packages/MDAnalysis/core/__init__.py in <module>
    463 from . import groups
    464 from . import selection
--> 465 from . import AtomGroup

~/anaconda3/lib/python3.7/site-packages/MDAnalysis/core/AtomGroup.py in <module>
     26 from .groups import (Atom, AtomGroup, Residue, ResidueGroup, Segment,
     27                      SegmentGroup)
---> 28 from . import universe
     29 
     30 

~/anaconda3/lib/python3.7/site-packages/MDAnalysis/core/universe.py in <module>
    126                      Atom, Residue, Segment,
    127                      AtomGroup, ResidueGroup, SegmentGroup)
--> 128 from .topology import Topology
    129 from .topologyattrs import AtomAttr, ResidueAttr, SegmentAttr
    130 

~/anaconda3/lib/python3.7/site-packages/MDAnalysis/core/topology.py in <module>
     62 import numpy as np
     63 
---> 64 from .topologyattrs import Atomindices, Resindices, Segindices
     65 from ..exceptions import NoDataError
     66 

~/anaconda3/lib/python3.7/site-packages/MDAnalysis/core/topologyattrs.py in <module>
     38 import Bio.Seq
     39 import Bio.SeqRecord
---> 40 import Bio.Alphabet
     41 from collections import defaultdict
     42 import copy

~/anaconda3/lib/python3.7/site-packages/Bio/Alphabet/__init__.py in <module>
     19 
     20 raise ImportError(
---> 21     "Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://biopython.org/wiki/Alphabet for more information."
     22 )

ImportError: Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://biopython.org/wiki/Alphabet for more information.


```

**Comment by IAlibay** [comment]
@PasinduT as detailed in #3050, the fix of this issue was implemented in version 1.0 of MDAnalysis. Could you try updating to 1.0 and seeing if you still face issues?

**Comment by PasinduT** [comment]
@IAlibay Yes, it worked. I guess conda just installs an old version of MDAnalysis by default. Thanks.
